browser/request_otr/sources.gni

```diff
@@ -3,25 +3,18 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this file,
 # You can obtain one at https://mozilla.org/MPL/2.0/.
 
-import("//brave/components/request_otr/common/buildflags/buildflags.gni")
+brave_browser_request_otr_sources = [
```

Comment:
do we still need sources.gni for this? I don't see anything that would create a circular dependency now